### ISSUE TITLE
Cannot disable motion detection

### ISSUE THREAD

**Issue by matej2** [ccrisan/motioneye#2189]
I was trying to disable motion detection (and command triger that is configured). On UI, it seems like it is disabled, but the command still triggers. I have managed to find relevant logs, I will provide more if needed:

```
Sep 15 18:41:07 raspberrypi systemd[1]: Started motionEye Server.
Sep 15 18:41:16 raspberrypi meyectl[268]:     INFO: hello! this is motionEye server 0.42.1
Sep 15 18:41:33 raspberrypi meyectl[268]:     INFO: cleanup started
Sep 15 18:41:33 raspberrypi meyectl[268]:     INFO: wsswitch started
Sep 15 18:41:33 raspberrypi meyectl[268]:     INFO: tasks started
Sep 15 18:41:33 raspberrypi meyectl[268]:     INFO: mjpg client garbage collector started
Sep 15 18:41:34 raspberrypi meyectl[268]:     INFO: server started
Sep 15 18:41:34 raspberrypi meyectl[268]:    ERROR: failed to disable motion detection for camera with id 1: Failed to connect to 127.0.0.1 port 7999: Connection refused
```

I hope I did provide enought information. Thanks for your help

**Comment by starbasessd** [comment]
How are you trying to disable motion detection?
WebGUI port 8765 or motion port 7999? If port 7999 what are you using for your command?

**Comment by matej2** [comment]
WebGUI. I was editing camera configuration

**Comment by starbasessd** [comment]
There are several different ways to 'disable' motion detection, each with different indications and results.
In Still Images, select 'Capture Mode: Manual'
In Motion Detection, Select Title Bar: Off'
Still Images does not disable detection completely.
Motion Detection DOES disable it completely, but only for a particular camera

If the motion web interface is enabled:
{IP}:{port}/{camid}/detection/start Start or resume motion detection.
{IP}:{port}/{camid}/detection/pause Pause the motion detection.
Send command: curl http://ip_address:7999/1/detection/pause
It will not show in the WebGUI.
More Info can be found here:
https://motion-project.github.io/motion_config.html#OptDetail_Webcontrol

**Comment by matej2** [comment]
Still images are completelly disabled, motion detection was disabled completelly. I have only one camera.

So I directly connected to device and ran curl to localhost. This worked. However, disabling it using UI does not work, and returns me the same error that I pasted in this issue.

**Comment by starbasessd** [comment]
I am confused. You state that you are trying to disable motion detection, but you also state it is already disabled, and yet it is still recording triggered motion?
Can you show Settings, Still Images, Settings, Movies, and Settings, Motion Detection, please? (Screen shots would be fine)

**Comment by matej2** [comment]
Yes, exactly. That is exactly what I wrote in issue description. This is the bug that i am trying to report.
![image](https://user-images.githubusercontent.com/11059438/133628903-fe469b91-5b1c-4ac0-b63d-0a8ae8ea7b24.png)
![image](https://user-images.githubusercontent.com/11059438/133629094-7cfd8bd6-6484-4784-9ab6-494b46093e93.png)

**Comment by starbasessd** [comment]
What do you have set in Settings, Movies?

**Comment by matej2** [comment]
![image](https://user-images.githubusercontent.com/11059438/133639336-b12dc78a-9358-4277-85eb-1d15f384d693.png)

**Comment by starbasessd** [comment]
OK, will build and look at it this afternoon.

**Comment by matej2** [comment]
I had some problems with OS so I reinstalled it and reproduced the issue. Here are some more details, hoping this would help:

- I installed Raspbian OS lite (no desktop)
- changed port from 8765 to 80 as defined here: https://github.com/ccrisan/motioneye/wiki/Configuration-File#port
- set up Dataplicity
- enabled camera (the one that is connected directly to pi)

I tried different configuration combinations, like enabling everything and then disabling motion detection. The error (Connection refused) still appears.

It appears that this is the relevant error: https://github.com/ccrisan/motioneye/blob/795616d15caa24fd35f1d41b37668086e1ce4049/motioneye/motionctl.py#L265

**Comment by starbasessd** [comment]
Before making any other changes, does the WebGUI work on 8765?
You can't open the Motion Detection section with it turned Off. If you can, you've broken it somewhere else.

**Comment by matej2** [comment]
I will check.

If you are talking about screenshot I posted earlier: Motion detection was turned off, I expanded it to take screenshot

**Comment by starbasessd** [comment]
When set to on on my machines, it opens the section
When set to off on my machines, it closes the section.
I cannot get it to stay open when set to off, in either motionEye on another OS or motionEyeOS.
What you posted for a screenshot should not be possible.

**Comment by matej2** [comment]
There are two screenshots

**Comment by matej2** [comment]
Solved, I created a screenshot with different app
